gen-webpack-config.js

```diff
@@ -0,0 +1,40 @@
+const path = require("path");
+
+module.exports = function(subPkgRoot) {
+    const subPkgJson = require(path.join(subPkgRoot, "./package.json"));
+    const subPkgDeps = subPkgJson.dependencies
+        ? Object.keys(subPkgJson.dependencies)
+        : [];
+
+    const rootPkgJson = require(path.join(subPkgRoot, "../../package.json"));
+    const rootPkgDeps = rootPkgJson.dependencies
+        ? Object.keys(rootPkgJson.dependencies)
+        : [];
+
+    return {
+        // TODO(kevinb): build both dev and prod
+        mode: "development",
+        entry: "./index.js",
+        output: {
+            libraryTarget: "commonjs2",
+            filename: "dist/index.js",
+            path: subPkgRoot,
+        },
+        externals: [...rootPkgDeps, ...subPkgDeps],
```

Comment:
The reason why we have to generate a separate webpack config for each sub-package is that we to exclude different dependencies from inclusion in `dist/index.js` for each module.